scripts/chief_deploy.py

```diff
@@ -51,6 +51,7 @@ def update_product_details(ctx):
 @task
 def update_locales(ctx):
     with ctx.lcd(os.path.join(settings.SRC_DIR, 'locale')):
+        ctx.local("dennis-cmd lint --errorsonly .")
```

Comment:
The ``set -e`` line means that if any command returns non-zero, the script exits immediately with non-zero. I'm not sure when ``msgfmt`` returns non-zero for a header issue, but it does for bad strings.